.github/scripts/update_tools.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+""""
+Update scanner versions in the install.py file.
+This script fetches the latest release versions of the specified scanners
+from their GitHub repositories and updates the version strings in the
+install.py file while preserving the original file format.
+"""
+
+import os
+import sys
+import re
+import requests
+
+# Add the root directory to the Python path
+script_dir = os.path.dirname(os.path.abspath(__file__))
+root_dir = os.path.dirname(os.path.dirname(script_dir))
+sys.path.insert(0, root_dir)
+
+# Now we can import from src
+from src.modules.install import PACKAGES  # pylint: disable=wrong-import-position
```

Comment:
Why ?

it's a bit ugly, either we install the lib with pip either you just add a symlink to the file in this directory as well
Or maybe you want to move this PACKAGES dict to a yaml and here you only update the yaml file and in the lib you read the yaml